Dockerfile.bindings

```diff
@@ -0,0 +1,33 @@
+FROM ubuntu:jammy
+
+RUN apt-get update -q && apt-get install -y git curl xz-utils tar gpg build-essential
```

Comment:
Should we pin the image to a SHA and avoid `apt-get update`? I'm wondering if we might run into differences because of that over time.

Reply:
I think apt-get update is unavoidable, default ubuntu docker image seems to have no state of apt repositories, even for basic stuff. I've pinned the image though.